### DISCUSSION TITLE
Error- Validation Data in Segnalazione

### DISCUSSION THREAD

**Discussion by lucamala04** [globaleaks/globaleaks-whistleblowing-software#3827] [Q&A]
Salve,
ho un problema con Globaleaks, ho installato il software.
Sembra funzionare correttamnete ma quando vado ad inserire una segnalazione, nel momento di premere invia, ricevo l'errore "Error - Validation Data".

non riesco a capire cosa devo fare.
Suggerimenti?
Grazie a tutti.

Buona giornata.

**Comment by lele1568** [comment]
Salve,

stesso problema.

Non riesco proprio a capire cosa devo fare, le impostazioni mi sembrano tutte corrette.

Versione del software: 4.13.21

Grazie a Tutti e Buona Giornata

**Comment by evilaliv3** [comment]
Vi suggerisco di verificare di aver assegnato almeno un ricevente ai vostri canali di segnalazione.

**Comment by paolo987** [comment]
Stesso problema anche io.
La cosa strana è che fino a 2gg fa la segnalazione non aveva problemi, ne ho avuto con la login con la discussion #3879 ma non vedo come sia collegata.
Per scrupolo sono andato nei Canali e verificato se c'erano dei riceventi e ci sono... l'ho rimosso ed inserito

![image](https://github.com/globaleaks/GlobaLeaks/assets/78037015/9832171b-1f76-445c-8725-963a1efd6088)

non riesco ad inviare segnalazioni e non capisco il perchè.


l'errore che compare è questo
![image](https://github.com/globaleaks/GlobaLeaks/assets/78037015/57c3c56a-9438-46d5-9afc-3d6411ee1b2c)

nei log nessun errore in particolare, se serve aumento il livello di verbosità
grazie del supporto @evilaliv3

**Reply by evilaliv3** [reply]
Ipotizzo che abbiate fatto un setup senza dare agli amministratrori le chiavi di recupero degli account e che per questo motivo siate in una condizione in cui gli account utente sono attivi solo dopo il primo login da parte degli utente.

Fate un primo login e riprovate.

**Comment by paolo987** [comment]
Si, hai ragione. 
Per la questione della #3879 avevo cancellato le chiavi su DB sperando risolvesse e questo evidentemente ha generato il problema.
Ho fatto accesso sia con l'utenza amministratore che come case manager e ottenuto le nuove chiavi.
rifatto il login e ora non ho problemi ad inviare segnalazioni.
Grazie del supporto.

**Comment by lele1568** [comment]
Ora Funziona,
effettivamente non avevamo assegnato il ricevente nei canali che gestiscono i modelli.
Grazie.
